lego-webapp/utils/achievementConstants.ts

```diff
@@ -92,21 +92,21 @@ export const AchievementsInfo: Record<
   ],
   event_rank: [
     {
-      name: 'Arrangement:\nMester',
+      name: 'Arrangementsridderen',
       description: '3. plass - flest arrangementer',
       rarity: 5,
       hidden: false,
       image: trofe_sjeldenhetsgrad_10,
     },
     {
-      name: 'Arrangement:\nStormester',
+      name: 'Arrangementsmonarken',
       description: '2. plass - flest arrangementer',
       rarity: 5,
       hidden: false,
       image: trofe_sjeldenhetsgrad_10,
     },
     {
-      name: 'Arrangement:\nFyrtårn',
+      name: 'Høye eksellense\nav arrangementer',
```

Comment:
Any reason this `\n` is used? If it is to make sure that the text wraps that could be more properly done with css